templates/layout.html

```diff
@@ -4,32 +4,105 @@
+                                    <li>
+                                        <a class="dropdown-item" id="archivedList"
+                                           href="{{ url_for('user.users', user_type='archived') }}">Archived users</a>
+                                    </li>
+                                    <li role="separator" class="dropdown-divider"></li>
+                                    <li>
+                                        <a class="dropdown-item" id="newUser"
+                                           href="{{ url_for('user.register') }}">New user</a>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li class="nav-item dropdown">
+                                <a class="nav-link dropdown-toggle" href="#" id="coursesDropdown" role="button"
+                                   data-bs-toggle="dropdown" aria-expanded="false">
+                                    Courses
+                                </a>
+                                <ul class="dropdown-menu" aria-labelledby="coursesDropdown">
+                                    <li>
+                                        <a class="dropdown-item" id="courseList"
+                                           href="{{ url_for('course.courses', current_year=dynamic_year) }}">Course
+                                            list</a>
+                                    </li>
+                                    <li role="separator" class="dropdown-divider"></li>
+                                    <li>
+                                        <a class="dropdown-item" id="newCourse"
+                                           href="{{ url_for('course.form_course') }}">New course</a>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li class="nav-item dropdown">
+                                <a class="nav-link dropdown-toggle" href="#" id="configDropdown" role="button"
+                                   data-bs-toggle="dropdown" aria-expanded="false">
+                                    Config
+                                </a>
+                                <ul class="dropdown-menu" aria-labelledby="configDropdown">
+                                    <li>
+                                        <button class="dropdown-item" data-bs-toggle="modal"
+                                                data-bs-target="#confirmationModal">Add New Year
+                                        </button>
+                                    </li>
+                                </ul>
+                            </li>
+                        {% endif %}
                         {% block additionalnav %}{% endblock %}
                     </ul>
                     <div class="navbar-text ms-auto">
+                        &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Bootstrap provides several CSS classes to play with margins and paddings. That may help to avoid putting non-breakable spaces.